tasks/install.yml

```diff
@@ -35,6 +35,12 @@
     - openvpn_use_ldap
     - ansible_distribution == "CentOS" and ansible_distribution_major_version == "8"
 
+- name: Compile OTP plugin
+  include_tasks: compile_otp_plugin.yml
+  when:
+    - openvpn_use_otp
+    - ansible_distribution == "CentOS"
```

Comment:
Is this strictly CentOS, or will it also run under Fedora with the epel repo available?

Reply:
I based this on the work that was done for the ldap plugin to be as similar as possible.